presto-client/src/main/java/io/prestosql/client/ClientTypeSignature.java

```diff
@@ -41,6 +43,7 @@
     private static final Pattern PATTERN = Pattern.compile(".*[<>,].*");
     private final String rawType;
     private final List<ClientTypeSignatureParameter> arguments;
+    public static final int UNBOUNDED_LENGTH = Integer.MAX_VALUE;
```

Comment:
"unbounded length" doesn't say what this is referring to, so:

```suggestion
    public static final int VARCHAR_UNBOUNDED_LENGTH = Integer.MAX_VALUE;
```

Reply:
This still wouldn't be great (unbounded varchar doesn't have length), but i don't currently have a better name, so can be "VARCHAR_UNBOUNDED_LENGTH".